### PR TITLE
Fixing oscap related issues according to 6.2

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_oscappolicy.py
S...
----------------------------------------------------------------------
Ran 4 tests in 745.940s

OK (SKIP=1)
```
```
nosetests tests/foreman/ui/test_oscapcontent.py
S...
----------------------------------------------------------------------
Ran 4 tests in 482.233s

OK (SKIP=1)
```

### CODE REVIEW

**Comment by oshtaier** [comment]
Re-opened BZ 1293296 as it was closed with a bad reason

File: tests/foreman/ui/test_oscapcontent.py [hunk 1]
```diff
@@ -76,27 +73,6 @@ def test_negative_create_with_invalid_name(self):
                     self.assertIsNotNone(session.nav.wait_until_element(
                         common_locators['haserror']))
 
-    @tier1
-    def test_positive_default(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_oscapcontent.py]: That case seems irrelevant for a huge amount of time. Of course, we can see mentioned content even on latest 6.1.x, but not on any of 6.2. As oscap plug-in is not even default one for 6.2, I see no reason to raise that question further.

Comment (sthirugn) [inline tests/foreman/ui/test_oscapcontent.py]: @kbidarkar you okay with this test removal?

**Comment by sthirugn** [comment]
ACK pending @kbidarkar review.

File: tests/foreman/ui/test_oscapcontent.py (hunk 1 above)
Comment (kbidarkar) [inline tests/foreman/ui/test_oscapcontent.py]: We still need this test-case. 
1) For sat6.2 steps have changed and we need to run a command "foreman-rake foreman_openscap:bulk_upload:default"
2) I have raised a bug and requested this to be part of the installer, https://bugzilla.redhat.com/show_bug.cgi?id=1336374

We have 2 options:
a) update automation-tools to run "foreman-rake foreman_openscap:bulk_upload:default"
b) add bugzilla check for the testcase, 1336374

**Comment by oshtaier** [comment]
Thanks @kbidarkar for your inputs

File: tests/foreman/ui/test_oscapcontent.py [hunk 2]
```diff
@@ -37,9 +34,9 @@ def test_positive_create(self):
         @Assert: Whether creating  content for OpenScap is successful.
         """
         with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
```
Comment (kbidarkar) [inline tests/foreman/ui/test_oscapcontent.py]: For Sat6.2 , we shouldn't be running this with "Any context". It should be running under an Org.

File: tests/foreman/ui/test_oscappolicy.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ def test_positive_create_with_policy_name(self):
                         content=content_name,
                         name=policy_name,
                         period=OSCAP_PERIOD['weekly'],
-                        profile=OSCAP_PROFILE['rhccp'],
+                        profile=OSCAP_PROFILE['c2s_rhel6'],
```
Comment (kbidarkar) [inline tests/foreman/ui/test_oscappolicy.py]: why these values have been changed ?

Comment (oshtaier) [inline tests/foreman/ui/test_oscappolicy.py]: Basically, values that has brackets doesn't work for all selenium and FF versions (https://github.com/seleniumhq/selenium/issues/674). If that is really matter to choose these specific values - just let me know and I will do a rollback as I am pretty sure that it should work on CI, but not on my env and some others too.

**Comment by kbidarkar** [comment]
ACK
